web-ng/src/app/components/layer-dialog/layer-dialog.component.scss

```diff
@@ -152,3 +154,12 @@
   padding: 7px 0;
   margin-right: 10px;
 }
+
+.horizontal-line{
+  border: 1px solid mat-color($light-grey);
```

Comment:
Rather than specifying a specific color, we should use an element from the theme defined in https://github.com/angular/components/blob/16f5f793dea3d83a6f08de14e3787ce97d82ff15/src/material/core/theming/_palette.scss#L674 (e.g., `$divider`).

This will allow us to update elements across the app in a consistent way, and to enable global skins like dark theme. 

Same comment throughout.